src/base/io-funcs.h

```diff
@@ -182,6 +182,17 @@ template<class T> inline void WriteIntegerVector(std::ostream &os, bool binary,
 template<class T> inline void ReadIntegerVector(std::istream &is, bool binary,
                                                 std::vector<T> *v);
 
+/// Function for writing array of integer types.
```

Comment:
Please just put these templates in const-arpa-lm.cc for now until we need them for something else later.  They are confusingly similar to ReadIntegerVector and WriteIntegerVector and I won't want people to assume these functions are the first choice for writing this type of thing.